addon/components/one-way-checkbox.js

```diff
@@ -15,29 +15,41 @@ const OneWayCheckboxComponent = Component.extend(DynamicAttributeBindings, {
   NON_ATTRIBUTE_BOUND_PROPS: ['update'],
 
   attributeBindings: [
-    'isChecked:checked',
     'type',
     'value'
   ],
 
+  checkedValue: computed('paramChecked', 'checked', function() {
+    let checkedValue = get(this, 'paramChecked');
+
+    if (checkedValue === undefined) {
+      return checkedValue = get(this, 'checked');
+    }
+
+    return checkedValue;
+  }),
```

Comment:
`checkedValue` is a computed property that amalgamates `checked` and `paramChecked` in a similar way the the previous iteration of code did inside `didReceiveAttrs`